openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/GlossaryAssets.spec.ts

```diff
@@ -0,0 +1,160 @@
+/*
+ *  Copyright 2025 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import test, { expect } from '@playwright/test';
+import { Glossary } from '../../support/glossary/Glossary';
+import { GlossaryTerm } from '../../support/glossary/GlossaryTerm';
+import { TableClass } from '../../support/entity/TableClass';
+import { DashboardClass } from '../../support/entity/DashboardClass';
+import { createNewPage, redirectToHomePage } from '../../utils/common';
+
+test.use({
+  storageState: 'playwright/.auth/admin.json',
+});
+
+test.describe('Glossary Asset Management', () => {
```

Comment:
This is duplicate of Add and Remove assets test in glossary.spec
We have a test where we are also checking for mutually exclusive case, which is not handled here.